libparsec/crates/platform_mountpoint/src/windows/mod.rs

```diff
@@ -0,0 +1,481 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+mod error;
+
+use anyhow::anyhow;
+use once_cell::sync::{Lazy, OnceCell};
+use std::{
+    ops::Deref,
+    path::{Path, PathBuf},
+    sync::{Arc, Mutex},
+};
+
+use winfsp_wrs::{
+    filetime_from_utc, filetime_now, u16cstr, CleanupFlags, CreateFileInfo, CreateOptions,
+    FileAccessRights, FileAttributes, FileInfo, FileSystem, FileSystemContext, PSecurityDescriptor,
+    Params, SecurityDescriptor, U16CStr, U16CString, VolumeInfo, VolumeParams,
+    WriteMode as WinWriteMode, NTSTATUS, STATUS_DIRECTORY_NOT_EMPTY, STATUS_NOT_IMPLEMENTED,
+    STATUS_RESOURCEMANAGER_READ_ONLY,
+};
+
+use libparsec_types::FileDescriptor;
+
+use crate::{EntryInfo, EntryInfoType, MountpointInterface, MountpointManager, WriteMode};
+
+static SD: Lazy<SecurityDescriptor> = Lazy::new(|| {
+    SecurityDescriptor::from_wstr(u16cstr!("O:BAG:BAD:P(A;;FA;;;SY)(A;;FA;;;BA)(A;;FA;;;WD)"))
+        .expect("unreachable, valid SecurityDescriptor")
+});
+
+static VOLUME: OnceCell<Arc<Mutex<VolumeInfo>>> = OnceCell::new();
+
+const ENTRY_INFO_EXTENSION: &str = "__parsec_entry_info__";
+const MAX_FILE_NODES: u64 = 1024;
```

Comment:
I don't understand those MAX_* constants...

1024 nodes means maximum 1024 files/folders ?
16 * 1024 * 1024 means maximum 16mo for a given file ?

are we back to the 1980's :trollface:  ?

Reply:
I removed and set `total_size` and `free_size` to `1TB` as it was